build.sbt

```diff
@@ -315,7 +316,7 @@ lazy val mimaSettings = Seq(
   mimaPreviousArtifacts := Set(organization.value %% moduleName.value % "2.3.3"),
   mimaBinaryIssueFilters := {
     // Macro internals - ignore
-    val macroFilters = List("CaseClassMacros", "IsCons1Macros", "LazyMacros")
+    val macroFilters = List("CaseClassMacros", "IsHCons1Macros", "IsCCons1Macros", "IsCons1Macros", "LazyMacros", "SingletonTypeMacros")
```

Comment:
I wonder if it's worth it to try and preserve binary compatibility here, I didn't make any effort